actions/promote/createBatch.js

```diff
@@ -111,7 +111,11 @@ async function main(params) {
 async function createBatch(batchManager, appConf) {
     const { sp } = await getConfig();
     const options = await getAuthorizedRequestOption({ method: 'GET' });
-    const promoteIgnoreList = appConf.getPromoteIgnorePaths();
+    let promoteIgnoreList = appConf.getPromoteIgnorePaths() || [];
+    if (promoteIgnoreList && typeof promoteIgnoreList === 'string') {
```

Comment:
I guess there is no harm to have this check and do the additional processing but wanted to know what caused the issue. AFAIK, the [UI](https://github.com/adobecom/milo/commit/9aa40899758ba8ae5a079e38b31662f7551e131f) is sending the list as an array. The API payload format has not changed with this. But LMK if that is not the case.

Reply:
Thanks @sukamat . Updated for both previous comment and this. As suggested the code handles covers both string and arrays